recipes/usher/build.sh

```diff
@@ -7,7 +7,7 @@ pushd build
 
 cmake -DTBB_DIR=${PWD}/../oneTBB-2019_U9 -DCMAKE_PREFIX_PATH=${PWD}/../oneTBB-2019_U9/cmake -DCMAKE_INSTALL_PREFIX=${PREFIX} ..
 
-make -j
+make -j 1
```

Comment:
```suggestion
make
```

Reply:
Is it ok to leave it explicit as we have it here. The problem is that -j4 and such do not work as we are running out of memory.